src/setup/config.rs

```diff
@@ -118,10 +135,10 @@ impl NodeMetaData {
                     ));
                 }
                 start_args.insert(n_args - 1, "--config".into());
-                start_args.insert(n_args, config_path.into_os_string());
+                start_args.insert(n_args, config_file_path.into_os_string());
             }
             NodeKind::Zcashd => {
-                start_args.push(format!("-conf={}", config_path.to_str().unwrap()).into());
+                start_args.push(format!("-datadir={}", config_path.to_str().unwrap()).into());
```

Comment:
What happens if the user adds his own `-datadir` arg? Which one wins?

Reply:
Any user defined `-datadir` arg will get ovveridden by Ziggurat's.

Reply:
This should be documented in the readme.